images/sponsors/adventurous-machines.svg

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This can be minified.

Reply:
This applies to the whole file. There's a lot of Inkscape specific stuff in here we don't need for web.